Elevel/Elevel.Api/Startup.cs

```diff
@@ -95,16 +95,18 @@ public void ConfigureServices(IServiceCollection services)
         }
 
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
-        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ApplicationDbContext context)
         {
             if (env.IsDevelopment())
             {
                 app.UseDeveloperExceptionPage();
-                app.UseSwagger();
-                app.UseSwaggerUI(c => c.SwaggerEndpoint("/swagger/v1/swagger.json", "Elevel.Api v1"));
             }
+            app.UseSwagger();
+            app.UseSwaggerUI(c => c.SwaggerEndpoint("/swagger/v1/swagger.json", "Elevel.Api v1"));
 
-            app.UseHttpsRedirection();
+            //app.UseHttpsRedirection();
+
+            context.Database.Migrate();
```

Comment:
will approve but you need to it to place where it will called once. Program.cs will be ideal I think